content/spin/v3/serverless-ai-api-guide.md

```diff
@@ -32,19 +32,19 @@ ai_models = ["codellama-instruct"]
 // -- snip --
 ```
 
-> Spin supports "llama2-chat" and "codellama-instruct" for inferencing and "all-minilm-l6-v2" for generating embeddings.
+> Spin supports the models of the Llama architecture for inferencing and "all-minilm-l6-v2" for generating embeddings.
 
 ### File Structure
 
-By default, the Spin framework will expect any already trained model files (which are configured as per the previous section) to be downloaded by the user and made available inside a `.spin/ai-models/` file path of a given application. For example:
+By default, the Spin framework will expect any already trained model files (which are configured as per the previous section) to be downloaded by the user and made available inside a `.spin/ai-models/` file path of a given application. The files for each of the models need to be placed under a folder named after the architecture of the model (i.e) for `llama2-chat`, the folder needs to be named `llama`. For example:
 
 ```bash
-code-generator-rs/.spin/ai-models/llama/codellama-instruct
+code-generator-rs/.spin/ai-models/llama/codellama-instruct/{safetensors | config.json | tokenizer.json}
```

Comment:
This feels less clear with the pseudo-file wildcards.  Maybe spell out a few files e.g.

```suggestion
code-generator-rs/.spin/ai-models/llama/codellama-instruct/safetensors
code-generator-rs/.spin/ai-models/llama/codellama-instruct/config.json
code-generator-rs/.spin/ai-models/llama/codellama-instruct/tokenizer.json
```

or use tree form or something like that?  Not sure what's best